flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -1024,9 +1024,25 @@ under the License.
 	<profiles>
 		<!-- Activate these profiles with -Phive-x.x.x to build and test against different Hive versions -->
 		<profile>
-			<id>hive-3.1.1</id>
+			<id>hive-3.1.2</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<!-- disable running test in parallel for hive 3.1.2, otherwise, there'll be some concurrency issues -->
+							<forkCount>1</forkCount>
+							<reuseForks>false</reuseForks>
```

Comment:
FYI this doesn't work because the settings in the unit/Itcase executions take precedence.

Reply:
The reuseForks setting does apply to unit tests, but the forkCount is genuinely ignored.

Reply:
Thanks for you explanation.